cmdstanpy/utils.py

```diff
@@ -125,16 +125,19 @@ def rdump(path: str, data: Dict) -> None:
             fd.write('\n')
 
 
-def check_csv(path: str) -> Dict:
+def check_csv(path: str, is_optimizing: bool = False) -> Dict:
     """Capture essential config, shape from stan_csv file."""
-    dict = scan_stan_csv(path)
+    dict = scan_stan_csv(path, is_optimizing=is_optimizing)
```

Comment:
I would use some other name than `dict`, but again it is not something to be worried about.